tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -34,6 +34,9 @@ def test_init():
     assert version_info.schema_version == SCHEMA_VERSION
     assert version_info.library_version == version.__version__
 
+    assert storage.get_current_version() == storage.get_head_version()
+    assert storage.get_all_versions() == [storage._get_base_version()]
```

Comment:
Could you tell me how to update this assertion when we add a new version like `v0.10.0.a`?
`storage._get_bese_version()` always returns `v0.9.0.a` which is the initial schema version managed by alembic, and I think we need to update the assertion as follows:
`assert storage.get_all_versions() == ['v0.10.0.a', storage._get_base_version()]`.

If so, the right-hand side of the equation seems a bit awkward, and I'd like to write all versions as string literals like `['v0.10.0.a', 'v0.9.0.a']`.